src/datadog/remote_config.cpp

```diff
@@ -138,18 +149,16 @@ void RemoteConfigurationManager::process_response(const nlohmann::json& json) {
 
       const auto& config_metadata =
           targets.at("/signed/targets"_json_pointer).at(config_path);
-      if (!contains(config_path, k_apm_product) ||
+      if (!contains(config_path, k_apm_product_path_substring) ||
```

Comment:
Idea from Caleb: Write a partial parser that reads the first path component, and based on its value either retrieves the appropriate component, or returns some "n/a" value.